schema.sql

```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
+CREATE TABLE vet_clinic (
```

Comment:
- [ ] Vet_clinic is meant to be the db name, not the table name, as per the requirements, you need to create a table name " `animals`, please update